src/orion/algo/space/configspace.py

```diff
@@ -70,20 +104,28 @@ def dimension(self, dim: Dimension) -> None:
 
     def real(self, dim: Real) -> FloatHyperparameter:
         """Convert a real dimension into a configspace equivalent"""
+        args = normalize_args(dim, dim.prior)
+
         if dim.prior_name in ("reciprocal", "uniform"):
-            a, b = dim._args
+            # NB: scipy uniform [loc, scale], configspace [min, max] with max = loc + scale, loc = min
+            if dim.prior_name == "uniform":
+                loc, scale = args
+                lower = loc
+                upper = loc + scale
+            else:
+                lower, upper = normalize_args(dim, dim.prior, ["a", "b"])
```

Comment:
Would using `lower, upper = dim.interval()` work instead? It would require less shenanigans if it could work. :thinking:

Reply:
running

Reply:
works